docs/scos/dev/feature-integration-guides/202200.0/multiple-abstract-products-promotions-and-discounts-feature-integration.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Multiple Abstract Products as Promotional Products & Discounts feature integration
+description: Add support of multiple abstract products as promotional products in the Promotions & Discounts feature.
+last_updated: Feb 04, 2022
+template: feature-integration-guide-template
+---
+
+This document describes how to add the *multiple abstract products as promotional products* to the [Promotions & Discounts](/docs/scos/user/features/{{page.version}}/promotions-discounts-feature-overview.html) feature.
+
+## Install feature core
+
+Follow the steps below to install the feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+| --- | --- | --- |
+| Promotions & Discounts | {{page.version}} | [Promotions & Discounts feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/promotions-and-discounts-feature-integration.html) |
+| Spryker Cart | {{page.version}}   | [Spryker Cart feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/cart-feature-integration.md.html) |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker/discount-promotions-rest-api "^1.4.0"
```

Comment:
that's not enough
at similar IG we updated feature

see https://github.com/spryker/spryker-docs/pull/684/files

please also discuss with Anton if you need to update basic IG as he did

Reply:
discussed